plaidml2/core/ffi.cc

```diff
@@ -51,6 +58,20 @@ void plaidml_init(plaidml_error* err) {
           el::Loggers::setVerboseLevel(level);
         }
       }
+      auto ctx = GlobalContext::getContext();
+      auto eventlog_str = vertexai::env::Get("PLAIDML_EVENTLOG_FILENAME");
+      if (ctx && eventlog_str.size()) {
+        IVLOG(1, "Logging events to " << eventlog_str);
+        auto config = "{\"@type\": \"type.vertex.ai/vertexai.eventing.file.proto.EventLog\", \"filename\": \"" +
+                      eventlog_str + "\"}";
```

Comment:
Can we use `llvm::formatv` for this instead?

Also, it'd be nice to use a raw C++ string literal so we don't have to escape the quotes.